cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/SortedReplicas.java

```diff
@@ -166,7 +166,7 @@ void add(Replica replica) {
    *
    * @param replica the replica to remove.
    */
-  void remove(Replica replica) {
+  public void remove(Replica replica) {
```

Comment:
Would you clarify the need for making `add` and `remove` `public`? If there's no specific reason, can we keep the existing level of privacy?